libs/CHANGELOG.md

```diff
@@ -4,6 +4,20 @@ All notable changes to this project will be documented in this file.
 
 ## [Unreleased]
 
+## [0.0.15]
```

Comment:
It seems 0.0.15 is not yet published? :eyes:

Reply:
That's correct. Afterwards, it was released yesterday. `@Logseq/libs` v0.0.15